code/game/objects/items/weapons/RFD.dm

```diff
@@ -1,3 +1,8 @@
+#define RFD_FLOORS_N_WALL 1
+#define RFD_WINDOWS_N_GRILLE 2
```

Comment:
At this point just put "AND" instead of "N", the latter gives me an aneurysm to look at